Makefile

```diff
@@ -11,6 +11,7 @@ NODE ?= node
 ZUUL ?= $(NODE) $(BIN)/zuul
 
 test:
+	$(BIN)/mocha test/*.js; \
```

Comment:
We now test in node, as well, using mocha, directly (as opposed to indirectly, through Zuul).